docs/contributing/index.md

```diff
@@ -70,52 +60,9 @@ The Ignite product team developers are focused on building Ignite CLI and improv
 
 Meet the [people behind Ignite CLI and our contributors](https://github.com/ignite-hq/cli/graphs/contributors).
 
-## Viewing tutorial builds
+## Viewing docs builds
 
 There are two ways to see what your changes will look like in production before the updated pages are published.
 
-- When a PR is ready for review, you can see a deployed preview on a URL that is unique for that PR.
-- While a PR is in draft mode, you can preview a local build.
-
-### Preview PRs on a deployed preview
```

Comment:
this functionality is no longer available to me, relies on the existing docs build, and is not heavily used, so I removed it